tailwind.base.css

```diff
@@ -157,6 +161,11 @@
     @apply font-medium text-gray-500 text-ellipsis break-words text-sm;
   }
 
+  /* TODO: rework centering in gn-ui-card-icon and maybe merge both classes into one */
+  .gn-ui-nav-bar-icon {
+    @apply gn-ui-card-icon justify-center items-center h-8 min-w-10 hover:bg-primary;
+  }
```

Comment:
this shouldn't be a new global class; global classes are for very generic elements like buttons, cards, input fields etc.

Reply:
Ok I'll try to use the `gn-ui-button` for this. I guess the `favorite-star` component can be wrapped into it.

Reply:
I refactored the toggle-star component to use the `gn-ui-button` and thus become "themable"